site/tests/app/controllers/admin/LibraryManageControllerTester.php

```diff
@@ -26,11 +28,12 @@ class LibraryManageControllerTester extends BaseUnitTest {
     /** @var LibraryManageController */
     protected $controller;
 
-    protected function createConfigWithLibrary(bool $enabled = true, string $location = 'library location') {
+    protected function createConfigWithLibrary(bool $enabled = true, string $location = 'library location', int $access_level = User::LEVEL_SUPERUSER) {
```

Comment:
I think this should have a default access level of guest so that we can get the tests to pass.